assets/js/lunr-feed.js

```diff
@@ -0,0 +1,96 @@
+---
+---
+{% if site.site_search %}
```

Comment:
@crisostomon I've encapsulated the search database with an if clause to not impact the jekyll page generation process.